tabu.sty

```diff
@@ -2041,8 +2094,8 @@
 }% \tabu@printdecimal
 %% Verbatim inside X columns ----------------------------------------
 \def\tabu@verbatim{%
-    \let\verb \tabu@verb
-    \let\FV@DefineCheckEnd \tabu@FV@DefineCheckEnd
+    % \let\verb \tabu@verb
+    % \let\FV@DefineCheckEnd \tabu@FV@DefineCheckEnd
```

Comment:
I was hoping my above patches meant no more hacking of `\verb` and fancyverb end of environment check was needed and it does seem to work!

So `\tabu@verbatim` could be entirely removed.